mkdocs.yml

```
@@ -123,6 +125,7 @@ extra:
   social:
     - icon: fontawesome/brands/twitter
       link: https://twitter.com/eessi_hpc
+  generated_time: "Wed, 07 Aug 2024 at 17:08:54 CEST"
```

I would add a comment here that this gets auto-updated via the script:
```suggestion
  # this gets auto-updated via update_generated_time.sh script run in update_available_software.yml action
  generated_time: "Wed, 07 Aug 2024 at 17:08:54 CEST"